### PR TITLE
Add BigQuery Drop Table capability

### PR DESCRIPTION
**Pull request by alogvinchuk-cv**

### CODE REVIEW

File: src/main/java/io/cdap/plugin/gcp/bigquery/actions/BigQueryDropTableAction.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+@Description("This action drops given BigQuery table.")
+public class BigQueryDropTableAction extends Action {
+  private BigQueryDropTableActionConfig config;
+
+  @Override
+  public void run(ActionContext actionContext) throws Exception {
+    config.validate();
+
+    String dataset = config.getDataset();
+    String tableNames = config.getTable();
```
Comment (nitinmotgi) [inline src/main/java/io/cdap/plugin/gcp/bigquery/actions/BigQueryDropTableAction.java]: tableName

File: src/main/java/io/cdap/plugin/gcp/bigquery/actions/BigQueryDropTableAction.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    String project = config.getDatasetProject();
+    Table table = BigQueryUtil.getBigQueryTable(config.getServiceAccountFilePath(), project, dataset, tableNames);
+
+    if (Objects.nonNull(table)) {
+      table.delete();
```
Comment (nitinmotgi) [inline src/main/java/io/cdap/plugin/gcp/bigquery/actions/BigQueryDropTableAction.java]: Does this throw an exception, if it does catch it and provide a simplified message.

File: widgets/BigQueryDropTable-action.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "display-name" : "Google Cloud BigQuery Drop Table",
```
Comment (nitinmotgi) [inline widgets/BigQueryDropTable-action.json]: BQ Drop Table. We are changing to short names.

File: widgets/BigQueryDropTable-action.json [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+                "id": "false",
+                "label": "Drop always"
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      "label" : "Credentials",
```
Comment (nitinmotgi) [inline widgets/BigQueryDropTable-action.json]: Advanced

File: widgets/BigQueryDropTable-action.json [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName",
+          "widget-attributes" : {
+            "placeholder": "Name used to identify this source for lineage"
+          }
+        },
+        {
+          "widget-type": "textbox",
```
Comment (nitinmotgi) [inline widgets/BigQueryDropTable-action.json]: Move this to "Advanced" section

File: widgets/BigQueryDropTable-action.json [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+          "label": "Project ID",
```
Comment (nitinmotgi) [inline widgets/BigQueryDropTable-action.json]: Project Id

File: widgets/BigQueryDropTable-action.json [hunk 7]
```diff
@@ -0,0 +1,78 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "display-name" : "Google Cloud BQ Drop Table",
```
Comment (nitinmotgi) [inline widgets/BigQueryDropTable-action.json]: BQ Drop Table

**Review by nitinmotgi — CHANGES_REQUESTED** [review]
Minor comments

**Comment by alogvinchuk-cv** [comment]
@nitinmotgi Is this desired look of configuration window?
image

**Comment by alogvinchuk-cv** [comment]
@nitinmotgi Is this desired look of configuration panel?
image

**Pull request closed without merging**